src/module/vue/components/buttons/btn-rollstat.vue

```diff
@@ -34,7 +33,7 @@ const $item = computed(() => {
 })
 
 function rollStat() {
-  console.log(props, $item.value)
+  // console.log(props, $item.value)
```

Comment:
Feel free to just delete this line, no need to keep it around but commented out.